linux_install.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Some environment path variables
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+MOZ_DEST="$HOME/.mozilla"
+JSON_DEST="$MOZ_DEST/native-messaging-hosts"
+OLD_PATH="/home/william/scripts/ff2mpv"
+
+# Copying the JSON
+if [ -d "$MOZ_DEST" ]; then
```

Comment:
Tiny nit: `shellcheck` doesn't catch this (I wish it would!), but `[[` is slightly faster and more expressive in `bash` than `[` (which actually requires an exec).